nss-evening-cohort-7/nutshell-flying-buttresses
# As style message buttons

TO test: 
1. Git fetch
2. git checkout as-styleMessageButtons
3. hs -o

If success:
The message buttons are not visible if you are not the author, if you are the author, they show up on mouseenter and disappear on mouseout. They are now the default button color
Username also appears in Message body instead of user id- unless that user is not entered in the user database
